extensions/vertx-http/deployment/src/test/java/io/quarkus/vertx/http/management/ManagementAndRootPathTest.java

```diff
@@ -2,9 +2,8 @@
 
 import java.util.function.Consumer;
 
-import javax.inject.Singleton;
-
 import jakarta.enterprise.event.Observes;
+import jakarta.inject.Singleton;
```

Comment:
@cescoffier To be honest - I don't understand what is the `MyObserver` used for?